examples/middleware/e2e/playwright.config.ts

```diff
@@ -49,5 +49,7 @@ export default defineConfig({
     command: "pnpm preview:worker",
     url: "http://localhost:8774",
     reuseExistingServer: !process.env.CI,
+    // the app takes a bit to boot up for some reason, that's why we need a longer timeout here
+    timeout: 2 * 60 * 1000,
```

Comment:
not too sure why this is needed now and it wasn't before 🤷

Reply:
ah.... probably because the boot up process takes a bit longer with `enableEdgeDevGetCloudflareContext`... weird the difference should be practically negligible (locally I don't notice any difference)